tests/unit/nupic/research/spatial_pooler_compatability_test.py

```diff
@@ -100,6 +100,8 @@ def compare(self, pySp, cppSp):
                            cppSp.getMinPctOverlapDutyCycles())
     self.assertAlmostEqual(pySp.getMinPctActiveDutyCycles(),
                            cppSp.getMinPctActiveDutyCycles())
+#    self.assertEqual(pySp.getRandomSP(),  #FIXME enable when done in C++ too
```

Comment:
Please create a new issue for addressing this (if there isn't one already), so it doesn't get lost.